### ISSUE TITLE
[docker] Hardcoded HTTP port in the generator container

### ISSUE THREAD

**Issue by tomtit** [swagger-api/swagger-codegen#3386]
As for [now](https://github.com/swagger-api/swagger-codegen/blob/bd7904d42a2a52afee169880f284b934ae45aa7d/modules/swagger-generator/Dockerfile#L11) generator container listens on port `8080`. To change the HTTP port I have to repeat the command from the `Dockerfile` with additional options:
`docker run -d swagger-generator java -jar /generator/jetty-runner.jar --port 80 /generator/swagger-generator.war`
`--port 80` is my addition to make container listen on port `80`
It would be great to an ability to set HTTP port via environment variables and don't depend on internals of the `Dockerfile`.

**Comment by kevinchabreck** [comment]
Couldn't you just map port 80 to the default swagger-generator port when creating your container?

`$ docker run -d -p 80:8080 swagger-generator`

**Comment by tomtit** [comment]
Nope, this is just a workaround. I don't want to expose any internal ports when I run `swagger-generator` as a part of the `docker-compose` project.
